pkg/jobrunaggregator/jobrunaggregatorlib/jobrun_locator_per_pr_jobs.go

```diff
@@ -9,12 +9,17 @@ import (
 	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
 )
 
-func GetAggregationIDFromProwJob(prowJob *prowjobv1.ProwJob) string {
-	return prowJob.Labels["release.openshift.io/aggregation-id"]
+const (
+	// AggregationIDLabel is the name of the label for the aggregation id in prow job
+	AggregationIDLabel = "release.openshift.io/aggregation-id"
+)
+
+func getMatchIDFromProwJob(prowJob *prowjobv1.ProwJob, label string) string {
```

Comment:
this method doesn't look useful

Reply:
Not for now. But this will be used later for matching either SheduleID or payload invocation ID. So it is useful.

Reply:
> Not for now. But this will be used later for matching either SheduleID or payload invocation ID. So it is useful.

A method that does nothing but lookup a map value isn't one we should have.

Reply:
When the map value was fixed, it made some sense.  But if you're passing the map value, it's not doing anythign for you.